src/common/auth/actions.js

```diff
@@ -44,3 +55,15 @@ export function login(fields) {
     }
   });
 }
+
+export function logout() {
+  return ({dispatch}) => {
+    dispatch(persistenceActions.remove('authToken'));
+
+    window.location.href = '/';
```

Comment:
Great comment. You guys should start with iOS React Native asap. Just try default Este app demo. All commits should be cooked with React Native in the mind.